models/blog.js

```diff
@@ -1,12 +1,12 @@
-
-const mongoose = require('mongoose')
+import mongoose from 'mongoose'
 const Schema = mongoose.Schema
 
 
 const articleSchema = new Schema({
   title: {
     type: String,
-    required: true
+    required: true,
+    unique: true
   },
   author: {
```

Comment:
It would be better if we link a blog with a user. Then it would make it easier to retrieve the information

## Achieve

You can set the type of a property to mongoose ObjectId types

Refer to this article here [Mongoose Foreign Relationship](https://vegibit.com/mongoose-relationships-tutorial/)

Reply:
thanks